.github/workflows/fork-ci.yaml

```diff
@@ -60,6 +60,7 @@ jobs:
     - uses: docker/setup-qemu-action@v3
     - uses: docker/setup-buildx-action@v3
     - uses: docker/login-action@v3
+    - uses: chrisdickinson/setup-yq@v1.0.1
```

Comment:
Have you tried running without this?
I believe GitHub's _ubuntu-latest_ runners have `yq` already available on them...